src/Grpc.Net.ClientFactory/GrpcHttpClientBuilderExtensions.cs

```diff
@@ -127,6 +136,18 @@ public static IHttpClientBuilder AddInterceptor(this IHttpClientBuilder builder,
         /// <param name="configureInvoker">A delegate that is used to create an <see cref="Interceptor"/>.</param>
         /// <returns>An <see cref="IHttpClientBuilder"/> that can be used to configure the client.</returns>
         public static IHttpClientBuilder AddInterceptor(this IHttpClientBuilder builder, Func<Interceptor> configureInvoker)
```

Comment:
Nit: document this has a `Channel` lifetime by default?

Reply:
Good idea.

I kind of want to rename `InterceptorLifetime`. It's more like a resolution scope. Someone could still register an interceptor as a singleton in DI and have a per-client interceptor be shared because it is a singleton in DI.